pages/api/table-pagination.md

```diff
@@ -28,7 +28,7 @@ A `TableCell` based component for placing inside `TableFooter` for pagination.
 | <span class="prop-name">nextIconButtonProps</span> | <span class="prop-type">object</span> |   | Properties applied to the next arrow [`IconButton`](/api/icon-button/) element. |
 | <span class="prop-name required">onChangePage *</span> | <span class="prop-type">func</span> |   | Callback fired when the page is changed.<br><br>**Signature:**<br>`function(event: object, page: number) => void`<br>*event:* The event source of the callback<br>*page:* The page selected |
 | <span class="prop-name">onChangeRowsPerPage</span> | <span class="prop-type">func</span> |   | Callback fired when the number of rows per page is changed.<br><br>**Signature:**<br>`function(event: object) => void`<br>*event:* The event source of the callback |
-| <span class="prop-name required">page *</span> | <span class="prop-type">number</span> |   | The zero-based index of the current page. |
+| <span class="prop-name">page</span> | <span class="prop-type">number</span> |   | The zero-based index of the current page. |
```

Comment:
Just noticed that our docs:api task does not pick up if a prop is required if at least one one validator that is chained is required.

Reply:
Already mentioned this, see above :wink:

Reply:
@eps1lon I'll try to resolve this